activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/behavior/MQServiceTaskBehavior.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.services.connectors.behavior;
+
+import java.util.Date;
+import java.util.UUID;
+
+import org.activiti.bpmn.model.ServiceTask;
+import org.activiti.engine.delegate.DelegateExecution;
+import org.activiti.engine.impl.bpmn.behavior.AbstractBpmnActivityBehavior;
+import org.activiti.engine.impl.delegate.TriggerableActivityBehavior;
+import org.activiti.engine.impl.persistence.entity.integration.IntegrationContextEntity;
+import org.activiti.engine.impl.persistence.entity.integration.IntegrationContextManager;
+import org.activiti.services.connectors.channel.ProcessEngineIntegrationChannels;
+import org.activiti.services.connectors.model.IntegrationEvent;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.support.MessageBuilder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MQServiceTaskBehavior extends AbstractBpmnActivityBehavior implements TriggerableActivityBehavior {
+
+    private static final String CONNECTOR_TYPE = "connectorType";
+    private final ProcessEngineIntegrationChannels channels;
+    private final IntegrationContextManager integrationContextManager;
+
+    @Autowired
+    public MQServiceTaskBehavior(ProcessEngineIntegrationChannels channels,
+                                 IntegrationContextManager integrationContextManager) {
+        this.channels = channels;
+        this.integrationContextManager = integrationContextManager;
+    }
+
+    @Override
+    public void execute(DelegateExecution execution) {
+        IntegrationContextEntity integrationContext = buildIntegrationContext(execution);
+        integrationContextManager.insert(integrationContext);
+
+        Message<IntegrationEvent> message = buildMessage(execution,
+                                                         integrationContext);
+        channels.integrationEventsProducer().send(message);
```

Comment:
@erdemedeiros At this point we need to send (maybe the same) an event to audit and probably query. As part of this task just make sure that you send the event, then we will define how to deal with it on different components.